### PR TITLE
feat: Print execution path as JSON array to prompt logs #168

### PR DESCRIPTION
**Pull request by astsiapanay**

### CODE REVIEW

File: src/main/java/com/epam/aidial/core/log/GfLogStore.java [hunk 1]
```diff
@@ -96,11 +99,10 @@ private void append(ProxyContext context, LogEntry entry) {
...
         }
 
-        String executionPath = context.getExecutionPath();
+        List<String> executionPath = context.getExecutionPath();
         if (executionPath != null) {
-            append(entry, ",\"execution_path\":\"", false);
-            append(entry, executionPath, true);
-            append(entry, "\"", false);
+            append(entry, ",\"execution_path\":", false);
+            append(entry, ProxyUtil.MAPPER.writeValueAsString(executionPath), true);
```
Comment (artsiomkorzun) [inline src/main/java/com/epam/aidial/core/log/GfLogStore.java]: do you want to see a json array in the log? append works only for a string

Comment (artsiomkorzun) [reply src/main/java/com/epam/aidial/core/log/GfLogStore.java]: ["x", "y"] -> willl be encoded as [""x"", ""y""]

Comment (astsiapanay) [reply src/main/java/com/epam/aidial/core/log/GfLogStore.java]: fixed

Comment (artsiomkorzun) [reply src/main/java/com/epam/aidial/core/log/GfLogStore.java]: ["x"y"] -> ["x"y"] - incorrect
["x"y"] -> ["x\"y"] - correct

You need to escape each element in a list.

**Review by artsiomkorzun — DISMISSED** [review]

**Review by artsiomkorzun — APPROVED** [review]

**Pull request merged**